README.md

```diff
@@ -61,6 +61,10 @@ the [community membership](https://github.com/koordinator-sh/community/blob/main
 
 Active communication channels:
 
+- Bi-weekly Community Meeting (APAC, *Chinese*):
+  - Tuesday 19:30 GMT+8 (Asia/Shanghai), [Calendar](dingtalk://dingtalkclient/action/open_mini_app?miniAppId=2018112162280005&ddMode=push&mainTask=true&keepAlive=false&newCalendar=1&page=pages%2Fdetail%2Findex%3FuniqueId%3DekNHQm9LdGMrTitUL0hWRWFYM3hXUT09%26recurrenceId%3D1669721400000%26inviter%3D248093415&fallback_url=dingtalk%3A%2F%2Fdingtalkclient%2Fpage%2Fcalendar_detail%3Fid%3D0%26uniqueId%3DekNHQm9LdGMrTitUL0hWRWFYM3hXUT09%26recurrenceId%3D1669721400000%26inviter%3D248093415)
```

Comment:
this link doesn't work.
![image](https://user-images.githubusercontent.com/24452340/203493266-bb74d5d1-bb69-465a-9f81-e852e14ae7dd.png)

Reply:
Oh, looks like Github markdown doesn't recognize `dingtalk://` scheme.

Reply:
@jasonliu747 temporarily remove the calendar, maybe we could create a google calendar later.